WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1200,8 +1200,12 @@ import WordPressComAnalytics
             }
 
             let failureBlock = { (error: Error?) in
+                guard let error = error else {
```

Comment:
It might make sense / be cleaner to move the cast so its part of the assigment.  Maybe something like:
```
guard let error = error as NSError! else {
        return
    }
```